cmd/podman/formats/formats.go

```diff
@@ -95,13 +97,20 @@ func (t StdoutTemplateArray) Out() error {
 	if err != nil {
 		return errors.Wrapf(err, "Template parsing error")
 	}
-	for _, img := range t.Output {
+	for i, img := range t.Output {
 		basicTmpl := tmpl.Funcs(basicFunctions)
 		err = basicTmpl.Execute(w, img)
 		if err != nil {
 			return err
 		}
-		fmt.Fprintln(w, "")
+		if i != len(t.Output)-1 {
+			fmt.Fprintln(w, "")
+			continue
+		}
+		// Only print new line at the end of the output if stdout is the terminal
+		if terminal.IsTerminal(int(os.Stdout.Fd())) {
```

Comment:
Shouldn't this be checking if w is a terminal instead of os.Stdout()? There is no guarantee w is os.Stdout()

Reply:
Won't w always write to stdout? And isn't it stdout that changes, it could be the terminal, or a file etc?
FWIW I tested this and it works as expected, but I could be wrong.

Reply:
You're right, nevermind - w is just a tabwriter always bound to stdout